apps/api/src/app/environments-v1/usecases/output-renderers/email-output-renderer.usecase.ts

```diff
@@ -245,12 +245,15 @@ export class EmailOutputRendererUsecase {
     });
   }
 
-  private stringToBoolean(value: unknown): boolean {
-    if (typeof value === 'string') {
-      return value.toLowerCase() === 'true';
-    }
+  private stringToBoolean(value: string): boolean {
+    const normalized = value.toLowerCase().trim();
```

Comment:
value.toLowerCase() could throw an exception if we value null/undefined/false/true and on everything that is not a string etc..

Reply:
It will always be a string, the function `stringToBoolean` accepts only string as an argument.

The reason is because `parseLiquid()` function returns always string:

```ts
  private async evaluateShowCondition(
    variables: FullPayloadForRender,
    node: MailyJSONContent & { attrs: { [MailyAttrsEnum.SHOW_IF_KEY]: string } }
  ): Promise<boolean> {
    const { [MailyAttrsEnum.SHOW_IF_KEY]: showIfKey } = node.attrs;
    const parsedShowIfValue = await this.parseLiquid(showIfKey, variables);

    return this.stringToBoolean(parsedShowIfValue);
  }
  ```